### ISSUE TITLE
Stop using example profiles as test fixtures

### ISSUE THREAD

**Issue by clintoncwolfe** [inspec/inspec#3351]
### 🎛 Description

🐛 bug report

In places like [test/functional/helper.rb](https://github.com/inspec/inspec/blob/master/test/functional/helper.rb), we store paths to test fixtures (testing resources on the filesystem, typically).  Many of these fixtures point to profiles that are in [examples/](https://github.com/inspec/inspec/blob/master/examples), which is a set of profiles intended to be good teaching examples.

While the needs of testing and teaching may overlap somewhat, they also diverge significantly.  For example, if you add controls to an example, that may cause a test that was relying the count of controls to fail.  Such coupling is harmful.

### 🌍 InSpec and Platform Version

2.2.78

### 💁 Possible Solutions

Initially, copy the example profiles into the test fixture area.  Consider stripping them down to the essential testing need, and / or consolidating them.

**Comment by clintoncwolfe** [comment]
Looks like we use things from Nathen's personal repo, too:

```
  1) Failure:
inspec exec::when using profiles on the supermarket#test_0002_can run supermarket profiles from inspec.yml [C:/projects/inspec/test/functional/inspec_exec_test.rb:359]:
Expected # encoding: UTF-8
"[2018-11-06T10:52:24+00:00] WARN: URL target https://github.com/nathenharvey/tmp_compliance_profile/ transformed to https://github.com/nathenharvey/tmp_compliance_profile/archive/master.tar.gz. Consider using the git fetcher\n" to include # encoding: UTF-8
"Profile Summary: \e[38;5;41m1 successful control\e[0m, \e[38;5;9m1 control failure\e[0m, 0 controls skipped\n".
```

**Comment by clintoncwolfe** [comment]
Refs #596

**Comment by clintoncwolfe** [comment]
Fixed by #3889
